compiler_opt/es/blackbox_optimizers.py

```diff
@@ -324,34 +471,6 @@ def set_state(self, state: SequenceOfFloats) -> None:
 
 See class descriptions for more detailed notes on each algorithm.
 """
-
-_GRAD_TYPE = flags.DEFINE_string('grad_type', 'MC', 'Gradient estimator.')
```

Comment:
can you do this removal in a separate PR? thanks.

Reply:
The removal has been separated into a different PR now